src/main/scala/verifier/DefaultMasterVerifier.scala

```diff
@@ -395,6 +398,12 @@ class DefaultMasterVerifier(config: Config, override val reporter: Reporter)
     sortWrapperDeclarationOrder foreach (component =>
       emitSortWrappers(component.sortsAfterAnalysis, sink))
 
+    val backendTypes = new mutable.HashSet[BackendType]

```

Comment:
Is that a deterministic collection? If not, we might be better of with a `LinkedHashset`, see https://github.com/viperproject/silicon/blob/master/src/main/scala/package.scala#L31.

Reply:
Done